Correction to Issue-19 has been implemented

hidetak opened the pull request:
* In the Consumer screen, the buttons have been modified to execute property write and action invoke
* Modified to display various invoke times on the consumer screen

hidetak committed:
In the Consumer screen, the buttons have been modified to execute property write and action invoke

egekorkan commented:
Thank you @hidetak for working on it! All my points in the linked issue are addressed however I have noticed a weird behavior. I have taken the td at http://plugfest.thingweb.io:8083/testthing and generated the flow. Everything but the following actions work: `int-void`, `int-int`, `int-string`. For those ones, I get nothing displayed on the UI but the flow editor shows internal server error. Sending the same request on postman, for example for `int-int` with input 12 returns 13. Action `void-obj` does exactly what I expect.

Also, for the actions with no output payload, can you add something to indicate that a response has been received? Maybe the same kind of timestamp would be nice.

The error is `Error: Server error: Internal Server Error`

hidetak committed:
fixed type conversion bug for action arguments and handling of cases with no return value

hidetak commented:
@egekorkan 

Thank you for the confirmation!

I apologize for the glitch.
The following fixes have been made.
* Fixed a bug in type conversion of action arguments
* Fixed to treat actions with no return value as null (also displays the timestamp when the return value was received)

There was one action that could not be performed.
When calling an obj-void action, the following error message was displayed.
```
The error is Error: Server error: Internal Server Error
```

The same error occurred when I tried the following simple test code
```
    const objvoidResult = await thing.invokeAction('obj-void', { “abc”: 123 })
```
The stack trace was as follows:
```
error: Error: Server error: Internal Server Error
    at HttpClient.checkFetchResponse (/Users/***/test-node-wot/test-node-wot/plugfest/node_modules/@node-wot/binding-http/src/ http-client-impl.ts:406:19)
    at HttpClient.<anonymous> (/Users/***/test-node-wot/test-node-wot/plugfest/node_modules/@node-wot/binding-http/src/http- client-impl.ts:196:14)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/***/test-node-wot/test-node-wot/plugfest/node_modules/@node-wot/binding-http/dist/http-client-impl.js:28 :58)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```
Do you know anything about it?

egekorkan commented:
@hidetak the changes work as expected, thank you! The internal server error is for sure with node-wot. Going to open an issue there. I am merging :)

Can you have a look at #23 and #24
